codeserver/ubi9-python-3.12/get_code_server_rpm.sh

```
@@ -0,0 +1,82 @@
+
+	# build libxkbfile
+	git clone https://gitlab.freedesktop.org/xorg/util/macros.git
+	cd macros/
+	./autogen.sh && make install -j ${MAX_JOBS}
+	export ACLOCAL_PATH=/usr/local/share/aclocal/
+	cd .. && rm -rf macros
+	git clone https://gitlab.freedesktop.org/xorg/lib/libxkbfile.git
+	cd libxkbfile/
+	./autogen.sh && make install -j ${MAX_JOBS}
+	cd .. && rm -rf libxkbfile
+        export PKG_CONFIG_PATH=$(find / -type d -name "pkgconfig" 2>/dev/null | tr '\n' ':')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)